src/index.js

```diff
@@ -23,7 +23,16 @@ Chart._adapters._date.override({
 	_create: function(time) {
 		return DateTime.fromMillis(time, this.options);
 	},
-
+	
+	/**
+	 * @private
+	 */
+	_isoWeekday: function(dateTime, weekday){
+		var normalizedWeekday = weekday % 7 ? weekday % 7 : 7;
```

Comment:
I think you could write this as:

    const normalizedWeekday = weekday % 7 || 7;

Reply:
done